items/contact3/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "Contact3",
+	"version": "1.0",
+	"release_date": "2018-10-07",
+	"license": "MIT",
+	"price_usd": "0",
+	"download_url": "https://github.com/novafacile/bludit-plugins/releases/download/contact3-v1/contact3.zip",
```

Comment:
the link for download is for Bludit v3 ? if so the field is called `download_url_v3`